package.json

```diff
@@ -79,8 +79,11 @@
     "cypress": "^8.6.0",
     "gh-pages": "^3.2.0",
     "parcel": "^2.3.1",
+    "process": "^0.11.10",
+    "punycode": "^1.4.1",
```

Comment:
seems like these are not used any where in the changes here?

Reply:
I don't remember adding these. You're right they are not needed. Removed and verified, everything is working as expected.